nav2_behavior_tree/plugins/action/remove_in_collision_goals_action.cpp

```diff
@@ -63,12 +63,18 @@ BT::NodeStatus RemoveInCollisionGoals::on_completion(
     return BT::NodeStatus::FAILURE;
   }
 
+  // get the `waypoint_statuses` vector if provided
+  std::vector<nav2_msgs::msg::WaypointStatus> waypoint_statuses;
```

Comment:
Then here you get the input as a string, and then find that in the blackboard directly to get/set